pyfftw/pyfftw.pyx

```diff
@@ -1394,6 +1405,10 @@ cdef class FFTW:
         copy the returned array.
         '''
 
+        if ortho and normalise_idft:
+            raise ValueError('Invalid options: ortho and normalise_idft cannot'
+                             ' both be True.')
+
```

Comment:
The `ortho` scaling is applied both to the dft and idft which makes the `normalise_idft` variable name slightly misleading.  I don't have too strong of an opinion either way, though.  Day to day I am usually using either the builders or the numpy interfaces so I don't call FFTW directly.